### PR TITLE
(Ticket Number) Add A Line For Kittens

### PR DESCRIPTION
**Pull request by mfb2**
Prior to this commit, the world was broken because reasons.
This commit adds unicorns and rainbows.  Maybe also cats.

### CODE REVIEW

File: my_email [hunk 1]
```diff
@@ -21,6 +21,7 @@ Editing ur text
 >^._.^<  Kitten
 >^._.^<  Kittens (2)
 >^._.^<  Kittens (3)
+>^._.^<  Kittens (4)
```
Comment (jeffweiss) [inline my_email]: This is indeed a kitten.

**Comment by mfb2** [comment]
Please approve this pull request because reasons.

File: my_email (hunk 1 above)
Comment (mfb2) [inline my_email]: Well, it's not really a kitten as much as it is a collection of kittens...  Minor syntactical clarification, really.

File: my_email [hunk 2]
```diff
@@ -21,6 +21,7 @@ Editing ur text
 >^._.^<  Kitten
```
Comment (mfb2) [inline my_email]: I think that @gchronis may want to weigh in here.  This is "kitten prime" and we need some context on this.

File: my_email (hunk 1 above)
Comment (gchronis) [inline my_email]: In any case I think that's quite enough kittens to be getting on with, at least for now. +1

**Pull request merged**